plugin.lua

```diff
@@ -1,185 +1,151 @@
-local function not_has(table, val)
-    for _, value in pairs(table) do
-        if value == val then
-            return false
-        end
-    end
-
-    return true
-end
+ITEM_WIDTH = 200
+WINDOW_WIDTH = 275
```

Comment:
increase window width by about 5 so that scroll bar doesn't cover part of the word "leniency"
```suggestion
WINDOW_WIDTH = 280
```